python/ray/_private/custom_types.py

```diff
@@ -72,9 +71,19 @@
     "DRIVER_TASK",
 ]
 TypeTaskType = Literal[tuple(TASK_TYPE)]
-TypeReferenceType = Literal[
-    tuple(reference_type.value for reference_type in ReferenceType)
+# TODO(kevin85421): `class ReferenceType(Enum)` is defined in
+# `dashboard/memory_utils.py` to avoid complex dependencies. I redefined
+# it here. Eventually, we should remove the one in `dashboard/memory_utils.py`
+# and define it under `ray/_private`.
+REFERENCE_TYPE = [
```

Comment:
Now we have two `REFERENCE_TYPE`s in our codebase, one for enum list, another for string list;
wondering if it's possible to combine them into one?

Reply:
NVM, see you TODO comment.